src/lucky/routable.cr

```diff
@@ -323,6 +322,26 @@ module Lucky::Routable
       Lucky::RouteHelper.new {{ method }}, path
     end
 
+    def self.route(*_args, **_named_args) : Lucky::RouteHelper
```

Comment:
Out of curiosity, if we just added `file = __FILE__, line = __LINE__` to this, would it pick up the correct file and line? It's a little hacky, but it would be nice to figure out how to get the location in here... There's also that weird `<#loc` thing in Crystal that I've always been curious about, but there's no docs on if it would even do what I'm thinking..

Reply:
I'm not sure what you actually want me to try here. I tried copy/pasting that line but it fails to compile

Reply:
oh, sorry.

```crystal
def self.route(*_args, _file = __FILE__, _line = __LINE__, **_named_args) : Lucky::RouteHelper
  puts "route at #{_file} #{_line}"
end
```

Though, after writing it, I realize this still won't get us the info at compile-time, so it's probably irrelevant 🤷‍♂️